SH4RE/SH4RE/Components/DropdownMenu.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  DropdownMenu.swift
+//  SH4RE
+//
+//  Created by Americo on 2023-01-22.
+//
+
+import SwiftUI
+
+
+struct DropdownStyle: DisclosureGroupStyle {
+    var noSelection: Bool
+    let button: some View = Image(systemName: "chevron.up")
+        .fontWeight(.medium)
+        .foregroundColor(.gray)
+    var border: some View {
+      RoundedRectangle(cornerRadius: 8)
+        .strokeBorder(
+            .gray,
+          lineWidth: 1
+        )
+    }
+    func makeBody(configuration: Configuration) -> some View {
+        ZStack (alignment: .topLeading){
+            if configuration.isExpanded {
+                VStack {
+                    //Spacer makes the drop down content visible from behind main label
+                    Spacer().frame(height: 52)
+                    configuration.content
+                }
+                
+                .disclosureGroupStyle(self)
+                .background(border)
+                .background(.white)
+                .cornerRadius(8)
+            }
+            HStack {
+                configuration.label
+                    .foregroundColor(noSelection ? .gray : .black)
+                Spacer()
+                button
+                    .rotationEffect(.degrees(configuration.isExpanded ? 180 : 0))
+            }
+            // Text alignment.
+            .multilineTextAlignment(.leading)
+            // Cursor color.
+            .accentColor(.primaryDark)
+            // Text color.
+            .foregroundColor(.black)
+            // TextField spacing.
+            .padding(.vertical, 16)
+            .padding(.horizontal, 20)
+            // TextField border
+            .background(border)
+            .background(.white)
+            .cornerRadius(8)
+            
+            
+            .contentShape(Rectangle())
+            .onTapGesture {
+                withAnimation {
+                    configuration.isExpanded.toggle()
+                }
+            }
+        }
+    }
+}
+
+struct DropdownMenu: View {
```

Comment:
I think the drop down arrow is backwards it should be pointing down when collapsed and up when open

Reply:
u rite, I change